src/foam/nanos/http/FileServlet.java

```diff
@@ -47,9 +47,13 @@ private void fileNotFoundError(HttpServletResponse resp, String file) {
 
   @Override
   protected void doGet(HttpServletRequest req, HttpServletResponse resp) {
-    String pathInfo = req.getPathInfo();
+    String pathInfo;
+    try {
+      pathInfo = req.getRequestURI();
+    } catch (Exception e) {
```

Comment:
UnsupportedOperationException, getReqquestURI throws it when launched using Java's built in Http server, while getPathInfo fails when using tomcat. I can switch it to be that error